src/suites/cts/command_buffer/copies.spec.ts

```diff
@@ -109,3 +109,67 @@ g.test('b2t2t2b', async t => {
 
   t.expectContents(dst, data);
 });
+
+g.test('imageBitmapToTexture', async t => {
+  const img = document.createElement('img');
+  img.src = '../../assets/img/test.png';
```

Comment:
`test.png` is pretty large. Can we choose a smaller one? E.g. make a simpler image with solid colors (which will compress better), and/or take one from here: 
https://github.com/KhronosGroup/WebGL/tree/master/sdk/tests/resources

Alternatively, we could just test only ImageBitmaps created from a `<canvas>` with generated contents, so we wouldn't need the PNG at all. This would make it easy to test lots (maybe 10s) of different image sizes, which would be really great.

Reply:
The only problem to test ImageBitmap from <canvas> is it won't cover uploading CPU data to WebGPU texture path (Pls correct me if I'm wrong, I think using canvas which has drawImage call means it is a textrue backed ImageBitmap). 
Happy to choose a smaller image for testing.

Reply:
That's true, yeah. If you are specifically trying to avoid texture-backed ImageBitmaps, you should be able to go through [ImageData](https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData): `createImageBitmap(new ImageData(uint8clampedarray, width))`

Reply:
Have you had a chance to evaluate whether this approach (ImageData from CPU) would work?

Reply:
Sry, it seems I missing all your comments when I push the wrong branch. Will try it today. Thanks for your suggestion.

Reply:
Tried ImageData from CPU and it works. So remove the image in latest update. Thanks for your guidance.